design/tls-client-verification.md

```diff
@@ -0,0 +1,125 @@
+# External client certificate validation (mutual TLS authentication)
```

Comment:
s/(mutual TLS authentication)//

Reply:
I have now removed references to mutual TLS authentication and mTLS.